src/daos/isRadosgwRunning.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
It's better to use the same naming conventions, i.e. `is_radosgw_running`

Reply:
I'll do that after the squash